app/lib/constants/site_config.rb

```diff
@@ -1,5 +1,7 @@
 module Constants
   module SiteConfig
+    IMAGE_PLACEHOLDER = "https://url/image.png".freeze
```

Comment:
I love this change! 😻 

Also love that this now sets the precedent for us to abstract _more_ of this file into constants, too!